src/src/AdminBundle/Resources/views/autocompleteModal.html.twig

```diff
@@ -0,0 +1,33 @@
+<div class="modal fade" id="{{ modal_id }}" tabindex="-1" role="dialog" aria-labelledby="assignModalLabel"
+     aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="assignModalLabel">{{ modal_title }}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <form method="post" action="{{ action_path }}">
+                <div class="modal-body">
+                    {{ modal_content }}
+
+                    <div class="ui-widget">
+                        <label for="birds">Birds: </label>
+                        <input name="birds" id="birds">
+                    </div>
+
+                    <div class="ui-widget" style="margin-top:2em; font-family:Arial">
```

Comment:
cely tento subor je divny. ako proof of concept je to fajn ale podla mna to ani nezodpoveda dizajnu z prototypu, ak s tym mas problem, napis, pomozem ti.

Reply:
nezodpoveda. som rada, ze som spravila aby to fungovalo. designa  drobnosti implementujem teraz.

Reply:
mam to teda prerobit?

Reply:
bud to prerob, alebo to zmaz a zajtra to spravime

Reply:
mam zmazat cely modal?? navrhujem nechat to tam a prerobit to v dalsom pr